src/main/java/salesforce/ui/pages/lightning/task/NewTaskLightPopUp.java

```diff
@@ -11,34 +11,39 @@
  *
  */
 
-package salesforce.ui.pages.lightning;
+package salesforce.ui.pages.lightning.task;
 
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.FindBy;
-import salesforce.ui.pages.abstracts.NewTaskAbstract;
+import salesforce.entities.Task;
+import salesforce.ui.pages.abstracts.task.NewTaskAbstract;
 
 /**
  * This class is for create a new class from light experience skin.
  * @author Melvi Caballero.
  * @version 0.0.1
  */
 public class NewTaskLightPopUp extends NewTaskAbstract {
+
     /**
      * Subject TextBox.
      */
     @FindBy(xpath = "//lightning-grouped-combobox[label[contains(text(),\"Subject\")]]/div/div/"
             + "lightning-base-combobox/div/div/input")
     private WebElement subjectTextBox;
+
     /**
      * save button.
      */
     @FindBy(xpath = "//button[@title=\"Save\"]")
     private WebElement saveButton;
+
     /**
      * notification close button.
```

Comment:
Review all of your comments and put at the beginning uppercase

Reply:
solved